### PR TITLE
Set rights

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem
- *Description of why you made this PR*

## Solution
- *And how do you fix that problem*

## PR Status
- [ ] Code finished.
- [ ] Tested with Package_check.
- [ ] Fix or enhancement tested.
- [ ] Upgrade from last version tested.
- [ ] Can be reviewed and tested.

## Package_check results
---
* An automatic package_check will be launch at https://ci-apps-dev.yunohost.org/, when you add a specific comment to your Pull Request: "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!"*

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

**Comment by yalh76** [comment]
!testme

File: scripts/upgrade [hunk 1]
```diff
@@ -78,10 +78,16 @@ if [ "$upgrade_type" == "UPGRADE_APP" ]
 then
 	ynh_script_progression --message="Upgrading source files..." --weight=6
 
+	ynh_secure_remove --dest_dir="$final_path"
```
Comment (yalh76) [inline scripts/upgrade:81]:
```suggestion
	ynh_secure_remove --file="$final_path"
```

**Review by yalh76 — CHANGES_REQUESTED** [review]

**Comment by ericgaspar** [comment]
!testme

File: scripts/upgrade [hunk 2]
```diff
@@ -78,10 +78,16 @@ if [ "$upgrade_type" == "UPGRADE_APP" ]
 then
 	ynh_script_progression --message="Upgrading source files..." --weight=6
 
+	ynh_secure_remove --file="$final_path"
```
Comment (yalh76) [inline scripts/upgrade:81]: you should backup and restore `/var/www/moodle/admin/cli/../../config.php` during source upgrade ^^

**Review by yalh76 — CHANGES_REQUESTED** [review]

**Pull request merged**